game.py

```diff
@@ -1,52 +1,23 @@
+import random
+from  wonderwords import RandomWord
+#https://pypi.org/project/wonderwords/
+
+SNOWMAN_GRAPHIC = [
+    '*   *   *  ',
+    ' *   _ *   ',
+    '   _[_]_ * ',
+    '  * (")    ',
+    '  \\( : )/ *',
+    '* (_ : _)  ',
+    '-----------'
+]
+
+SNOWMAN_WRONG_GUESSES = len(SNOWMAN_GRAPHIC)
```

Comment:
We could use this instead of `SNOWMAN_MAX_WRONG_GUESSES` later in the code, since this is related directly to the actual snowman image data. Or, we could use this to initialize `SNOWMAN_MAX_WRONG_GUESSES` to minimize changes later in the code.